pages/getting-started/quick-start/generate.mdx

```diff
@@ -1,41 +1,23 @@
-import { Callout, Tabs, Tab } from "nextra/components"
-import { TutorialSchema } from "../../../components/TutorialSchema"
+import { Callout, Tabs, Tab } from 'nextra/components';
+import { TutorialSchema } from '../../../components/TutorialSchema';
 
 # Generate Quick Start Guide
```

Comment:
We still need to decide what we want to call this thing now, and if/how it matches `@snaplet/client`. Planning on bringing it up in a Monday call.